workers/unity/Packages/com.improbable.gdk.core/Utility/EntityBuilder.cs

```diff
@@ -34,6 +41,15 @@ private EntityBuilder()
             entity = new Entity();
         }
 
+        /// <summary>
+        ///     Adds an arbitrary SpatialOS component with data to the entity template.
```

Comment:
Adds the provided spatialos component data. Adds an arbitrary makes it like like it picks one at random

Reply:
That would be quite funny tbf